images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -475,7 +529,7 @@ meteringconfig_reporting_overrides:
           enabled: "{{ meteringconfig_reporting_enable_post_kube_1_14_datasources }}"
 
 # combine the _meteringconfig_tls_overrides dictionary last to enforce when spec.tls.enabled is specified and set to true
-meteringconfig_spec: "{{ meteringconfig_default_values | combine(meteringconfig_default_image_overrides, meteringconfig_storage_overrides, meteringconfig_reporting_overrides, meteringconfig_spec_overrides, _meteringconfig_root_ca_overrides, _meteringconfig_tls_overrides, recursive=True) }}"
+meteringconfig_spec: "{{ meteringconfig_default_values | combine(meteringconfig_default_image_overrides, meteringconfig_storage_overrides, meteringconfig_reporting_overrides, _meteringconfig_ocp_disabled_overrides, meteringconfig_spec_overrides, _meteringconfig_root_ca_overrides, _meteringconfig_tls_overrides, recursive=True) }}"
```

Comment:
I put the ```_meteringconfig_ocp_disabled_overrides``` dictionary before the ```meteringconfig_spec_overrides``` to handle overriding the default Prometheus URL set in early role execution in a `MeteringConfig` configuration.

Reply:
Does TLS overrides coming later have any potential issues?

Reply:
Oh I see above, I'm not sure that I'm a fan of this approach since it's a bit harder to fully understand, but let me think on it.